test/saithrift/tests/sail3.py

```diff
@@ -807,6 +807,244 @@ def runTest(self):
+                                           ip_src=ip_source,
+                                           tcp_sport=source_port,
+                                           tcp_dport=destination_port,
+                                           ip_id=106,
+                                           ip_ttl=64) 
+                #packet at port1 
+                expected_packet1 = simple_tcp_packet(eth_dst='00:11:22:33:44:51',   
+                                                     eth_src=router_mac,
+                                                     ip_dst= ip_destination,
+                                                     ip_src=ip_source,
+                                                     tcp_sport=source_port,
+                                                     tcp_dport=destination_port,
+                                                     ip_id=106,
+                                                     ip_ttl=63)  
+                #packet at port2 
+                expected_packet2 = simple_tcp_packet(eth_dst='00:11:22:33:44:52',
+                                                     eth_src = router_mac,     
+                                                     ip_dst= ip_destination,
+                                                     ip_src=ip_source,
+                                                     tcp_sport=source_port,
+                                                     tcp_dport=destination_port,   
+                                                     ip_id =106,       
+                                                     ip_ttl=63) 
+                #packet at port3  
+                expected_packet3 = simple_tcp_packet(eth_dst='00:11:22:33:44:53',
+                                                     eth_src = router_mac,     
+                                                     ip_dst= ip_destination,
+                                                     ip_src=ip_source,
+                                                     tcp_sport=source_port,
+                                                     tcp_dport=destination_port,     
+                                                     ip_id =106,       
+                                                     ip_ttl=63) 
+
+
+                send_packet(self, 3, str(packet)) 
+                rcv_idx = verify_any_packet_any_port(self, [expected_packet1, expected_packet2, expected_packet3], [0, 1, 2])
+                count[rcv_idx] += 1
+                source_port += 1
+                destination_port += 1
+            
+            print "Packet distribution After changing HashSeed value"
+
+            for i in range(0,3):
+                print ("packets received at port interface : " + i)  
+                print (count[i])
+                if (count[i] >= ( maximum_packets/ 3) * 0.8):
+                    print "Not all paths are equally balanced, %s" % count
+                print "ALL paths are balanced with three members"    
+	
+	    finally:
```

Comment:
Please fix the indentation for 'finally'

Reply:
changes are made as per comments